README.md

```diff
@@ -64,15 +64,26 @@ Also check out [@ProjectAntrea](https://twitter.com/ProjectAntrea) on Twitter!
 
 ## Features
 
-Antrea currently supports the following features:
-* IPv4 overlay network for a Kubernetes cluster. Geneve, VXLAN, GRE, or STT can
-be used as the encapsulation protocol.
-* [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies)
-implementation.
-* [Octant](https://github.com/vmware-tanzu/octant) UI plugin for monitoring
-Antrea components, which publish runtime information as
-[CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
-* [IPsec encyption](/docs/ipsec-tunnel.md) of GRE tunnel traffic.
+* **Kubernetes-native**: Antrea follows best practices to extend the Kubernetes
```

Comment:
this is much better! any chance we can also include our Antrea proxy leveraging ovs?

Reply:
I was considering adding a "**Powered by Open vSwitch**" section and mention Antrea Proxy there. If others think it's a good idea, I can include it.

Reply:
I think good to add some points with OVS too.